src/Raven.Client/Documents/Commands/Batches/PutCommandDataBase.cs

```diff
@@ -20,8 +20,8 @@ public override void OnBeforeSaveChanges(InMemoryDocumentSessionOperations sessi
 
     public class PutCommandData : PutCommandDataBase<DynamicJsonValue>
     {
-        public PutCommandData(string id, string changeVector, DynamicJsonValue document)
-            : base(id, changeVector, document)
+        public PutCommandData(string id, string changeVector, DynamicJsonValue document, ForceRevisionStrategy strategy = ForceRevisionStrategy.None)
```

Comment:
We need to create a 2nd ctor to avoid issues with binary compatibility

Reply:
done
https://github.com/ravendb/ravendb/pull/9327/commits/a283f2ad28424f4f2bc547234d610be460e7ad19